### ISSUE TITLE
Nginx returns 502 after install script finishes

### ISSUE THREAD

**Issue by gbp** [mysociety/alaveteli#6057]
We've had report that install-site.sh is broken, EG:

```bash
    curl -O https://raw.githubusercontent.com/mysociety/commonlib/master/bin/install-site.sh
    sudo sh install-site.sh --default alaveteli alaveteli alaveteli.127.0.0.1.xip.io
```

We should expect the application to start, instead it appears we're seeing a 502 returned by nginx.

**Comment by jumus-jumbuck** [comment]
I am having the same issue, I did get a basic version of the site yesterday but I have been unable to get there again and I have tested multiple distros

**Comment by jumus-jumbuck** [comment]
I would really like to get a working version live so I can see if it is right for me

**Comment by jumus-jumbuck** [comment]
I am going to have a pop at updating the script, if it is worthy I will try and submit it because I have a project I am really keen to get launched

**Comment by gbp** [comment]
This should be fixed in [0.43](https://github.com/mysociety/alaveteli/blob/develop/doc/CHANGES.md#04300)
